tests/ModelContextProtocol.Tests/Protocol/PrimitiveSchemaDefinitionTests.cs

```diff
@@ -270,37 +223,6 @@ public static void NumberSchema_EmptyUnknownObject_IsIgnored()
         Assert.Equal(100.0, numberSchema.Maximum);
     }
 
-    [Fact]
-    public static void EnumSchema_UnknownPropertiesBetweenRequired_AreIgnored()
```

Comment:
This test can remain, removing the `enumNames` data and references, and asserting that it deserializes to an `UntitledSingleSelectEnumSchema`.